docs/api.rst

```diff
@@ -3132,7 +3132,7 @@ Search for assets(Levenshtein)
    :resjson string name: Name of the asset.
    :resjson string symbol: Symbol of the asset.
    :resjson string evm_chain: This value might not be included in all the results. Full name of the EVM chain where the asset is located if the asset is an EVM token.
-   :resjson string custom_asset_type: This value might not be included in all the results. It represents the custom asset type for a custom asset.
+   :resjson string asset_type: This value might not be included in all the results. It represents the asset type. Can be `custom asset`, `nft`, etc.
    :resjson bool is_custom_asset: A boolean to represent whether the asset is a custom asset or not.
```

Comment:
The part of the request is missing the arguments for the filters of `include/exclude` that you added right, also the `search_nfts`?